Dockerfile

```diff
@@ -85,3 +85,7 @@ COPY --from=trivy-scanner-build /workspace/out/trivy-scanner /
 USER 65532:65532
 WORKDIR /var/lib/trivy
 ENTRYPOINT ["/trivy-scanner"]
+
+FROM scratch as non-vulnerable
+COPY . .
+ENTRYPOINT [ "/non-vulnerable" ]
```

Comment:
is this needed? i don't think it exists?

Reply:
removed